supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc

```diff
@@ -427,7 +427,7 @@ The practice of having both modes together is often referred to as "hybrid", "ag
 [discrete]
 [[build-configuration]]
 ==== image:images/yes.png[yes] build config (noun)
-*Description*: In Red Hat OpenShift, a build config describes a single build definition and a set of triggers for when a new build should be created. The API object for a build config is `BuildConfig`.
+*Description*: In Red Hat OpenShift, a _build config_ describes a single build definition and a set of triggers for when a new build should be created. The API object for a build config is _BuildConfig_.
```

Comment:
Would you mind putting the last `BuildConfig` in backticks. (consistent with other openshift terms that do the same)

Reply:
:white_check_mark: